projects/gnomad/src/client/routes.js

```diff
@@ -21,6 +21,9 @@ import RegionPageContainer from './RegionPage/RegionPageContainer'
 import TranscriptPageContainer from './TranscriptPage/TranscriptPageContainer'
 import VariantPage from './VariantPage/VariantPage'
 
+import Logout from './Logout'
```

Comment:
To keep things clean I would do 
```suggestion
import LogoutPage from './ChorusPages/Logout'
```

Reply:
Done on next commit